data/distro.yml

```diff
@@ -477,3 +477,21 @@
         <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
       </li>
     </ol>
+    
+ - name: "GNU Guix"
+  logo: "guix.svg"
+  info: >
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>Flatpak can be installed using GNU Guix's default repositories:</p>
+        <pre><code>
+          <span class="unselectable">$</span> # guix install flatpak
```

Comment:
I have just noticed `<span class="unselectable">$</span> #`. This is wrong, it should be `<span class="unselectable">#</span>`.

I will be happy to merge this PR after this is fixed and [my previous question](https://github.com/flatpak/flatpak.github.io/pull/337/files#r321697811) is answered. ;-)

Reply:
See the suggested changes.